foxy/distribution.yaml

```diff
@@ -6974,6 +6974,21 @@ repositories:
       url: https://github.com/lagadic/visp.git
       version: master
     status: maintained
+  vrpn:
+    doc:
+      type: git
+      url: https://github.com/vrpn/vrpn.git
+      version: version_07.35
```

Comment:
@alvinsunyixiao IIRC version shouldn't be a tag.

Reply:
Sorry, I am releasing for the first time, so I might be wrong, but I saw the prompt when doing `bloom-release` and it says that this version could be a branch **or** a tag.

Reply:
For doc it can be either, for source it has to be a branch. But for doc we strongly recommend the branch so that it gets updates when you fix things on the branch instead of pointing to the old tag.